### DISCUSSION TITLE
Display slow interface top N？

### DISCUSSION THREAD

**Discussion by FootyMan** [apache/skywalking#11197] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Can there be 20 and 30 parameter settings for the default display of 10 slow interfaces and instances on the page?

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Isn't it already there through MQE expression? Have you checked the latest release?

**Reply by FootyMan** [reply]
What is version 9.3.0 and MQE used?

**Reply by wu-sheng** [reply]
If you want to check old release, it is on your own. Changelogs are there.

**Reply by FootyMan** [reply]
Not brother, what I'm asking is that there are 10 slow interfaces displayed on the page. Is there a parameter setting of 20?
<img width="548" alt="image" src="https://github.com/apache/skywalking/assets/9363830/f99cb8a6-c078-4d53-b592-8455620bda39">

**Reply by FootyMan** [reply]
Can the number of displays be dynamically set

**Reply by wankai123** [reply, marked as answer]
If the old version without MQE can refer:
<img width="1176" alt="image" src="https://github.com/apache/skywalking/assets/16773043/6403add7-4e70-4bbe-8646-f68af4c565f2">
